packages/kbn-es-query/package.json

```diff
@@ -1,14 +1,17 @@
 {
   "name": "@kbn/es-query",
-  "main": "target/index.js",
+  "main": "target/server/index.js",
+  "browser": "target/public/index.js",
   "version": "1.0.0",
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "build": "tsc && babel src --out-dir target",
+    "build": "rm -rf target && yarn build:server && yarn build:browser",
```

Comment:
Is the `rm -rf` portion really necessary? Wouldn't a `yarn kbn clean` do the trick?

Reply:
IMO, the build shouldn't include extra files when it's done, but a rebuild after renaming a file would do just that.